src/main.rs

```diff
@@ -143,6 +143,17 @@ fn launch_waiting_gordo_workflows(
     }
 }
 
+// Get a minor version from standard SemVer string
+fn minor_version(deploy_version: &str) -> Option<u32> {
+    deploy_version
+        .split('.')
+        .skip(1)
+        .take(1)
+        .map(|v| v.parse::<u32>().ok())
+        .last()
+        .unwrap_or(None)
```

Comment:
does it not make sense to have this as `unwrap_or(0)` instead? Then if it cant parse the version for some reason it will fallback to the old method, and I guess the check `if minor_version(&gordo.spec.deploy_version) >= Some(33) ` has less chance of failing?